orcoursetrion/tests/test_core.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+Test core functionality
+"""
+import unittest
+
+import semantic_version
+
+
+class TestCore(unittest.TestCase):
+    """Test common and core components to the package.
+    """
+
+    def test_version(self):
+        """Validate version matches proper format"""
+        # Will raise ValueError if not a semantic version
+        import orcoursetrion
+        semantic_version.Version(orcoursetrion.VERSION)
```

Comment:
That means you end up with uncovered test code which I disklike, so I prefer to let it raise rather than forcibly fail if raised. That said I've gone back and force on the subject.